plugins/sanity-plugin/gatsby-node.js

```diff
@@ -91,7 +91,7 @@ exports.createSchemaCustomization = async ({ actions }) => {
     interface HomepageImage implements Node {
       id: ID!
       alt: String
-      gatsbyImageData: JSON
+      gatsbyImageData: GatsbyImageData
```

Comment:
Honestly still a little confused as to why only the `sanity-plugin` is affected by this type change. Best theory is that `gatsby-source-sanity` includes the `graphql-typegen` feature in it, and thus we expect `GatsbyImageData` rather than the generic `JSON`